docs/_data/sidebars/left-navigation-sidebar.yml

```diff
@@ -72,6 +72,10 @@ entries:
       url: /components/alert.html
       output: web
 
+    - title: Asset Upload
+      url: /components/asset-upload.html
+      output: web
+
```

Comment:
When on the `http://localhost:4000/components/index.html` page, I am not seeing **Asset Upload** in the left nav, however, if I navigate to a component page (_e.g._ **Alert**), then I do see it.

Reply:
I don't see this problem. maybe the docs site was not generated with the latest? 

<img width="1435" alt="Screen Shot 2019-07-22 at 12 07 36 PM" src="https://user-images.githubusercontent.com/15215400/61650507-7ece9980-ac79-11e9-8b9b-3dd15cc94b3f.png">